src/esp/gfx/test/DepthUnprojectionTest.cpp

```diff
@@ -325,7 +326,7 @@ void DepthUnprojectionTest::benchmarkGpuDirect() {
       .setProjectionMatrix(
           Mn::Matrix4::perspectiveProjection(60.0_degf, 1.0f, 0.001f, 100.0f));
 
-  CORRADE_BENCHMARK(10) { mesh.draw(shader); }
+  CORRADE_BENCHMARK(10) { shader.draw(mesh); }
```

Comment:
This is great. `shader.draw(mesh)` is more intuitive than the old one.
I had this thought ever the beginning when I used Magnum library.

Reply:
You're not alone -- and you should have told me that, I'm blind to past mistakes ;) 

And as far as I asked everyone else who thought the same didn't complain because they thought "there has to be a reason why it's like that" (there wasn't, it happened to be this way because of a sequence of unrelated events). Fortunately one user finally had the courage to point this out, and I happily changed for the better.